### PR TITLE
Feature/600 sri zkp component implement zk circuit - Part 3

### PR DESCRIPTION
**Pull request by biscuitdey**
# Description

This PR adds script for generating circuit proving & verification keys. It also adds a placeholder circuit.

## Related Issue

#600

## How Has This Been Tested

Running the command generates the required files.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"solidity.compileUsingRemoteVersion": "v0.6.11+commit.5ef660b1"
```
Comment (Therecanbeonlyone1969) [inline .vscode/settings.json:2]: current version  is 0.8.18. Why not the latest version?

**Review by Therecanbeonlyone1969 — COMMENTED** [review]
LGTM ... left one question on compiler versioning. good to merge after that is answered.

**Review by Kasshern — APPROVED** [review]
Looks good given what has been pushed up. Approving now but still pending other's questions.

File: .vscode/settings.json (hunk 1 above)
Comment (biscuitdey) [reply .vscode/settings.json:2]: Snarkjs version 0.5.0 (stable version) automatically generates the solidity contract and uses the 0.6.11 compiler version. The latest snarkjs version (0.7.0) uses compiler version 0.8.18 but the code is unstable.

**Review by Therecanbeonlyone1969 — APPROVED** [review]
LGTM

**Pull request merged**